src/store/keyword.js

```diff
@@ -0,0 +1,107 @@
+import { keywordAPI } from "api";
+import { createAction,handleActions } from "redux-actions";
+import createRequestSaga,{createRequestSagaActionTypes} from "./createRequestSaga";
+import { takeLatest } from "redux-saga/effects";
+
+const [INQUIRY,INQUIRY_SUCCESS,INQUIRY_FAILURE] = createRequestSagaActionTypes("keyword/INQUIRY");
+const [GETKEYWORDLIST,GETKEYWORDLIST_SUCCESS,GETKEYWORDLIST_FAILURE] = createRequestSagaActionTypes("keyword/GETKEYWORDLIST");
```

Comment:
허허... underscore 없으니 가독성이 정말 안좋네요

```suggestion
const [GET_KEYWORD_LIST,GET_KEYWORD_LIST_SUCCESS,GET_KEYWORD_LIST_FAILURE] = createRequestSagaActionTypes("keyword/GETKEYWORDLIST");
```

Reply:
underscore 추가하도록 하겠습니다!

Reply:
8a3e07e